corehq/pillows/sofabed.py

```diff
@@ -22,8 +23,12 @@ def process_sql(self, doc_dict, delete=False):
                 pass
             return
 
+        if doc_dict.get('xmlns') == DEVICE_LOG_XMLNS:
```

Comment:
fyi, you shouldn't have to worry about this case anymore since device logs are now being saved straight to SQL